lib/WPA/FlowSensitive.cpp

```diff
@@ -78,8 +78,8 @@ void FlowSensitive::analyze()
 
         callGraphSCC->find();
 
-        solve();
-
+        initWorklist();
+        solveWorklist();
     }
```

Comment:
Which `init`/`solveWorklist` functions are called here, being that `FlowSensitive` inherits from `WPAFSSolver`, instead of `WPASolver`.